Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM nginx:alpine
+
+RUN apk --no-cache add nodejs-lts git
+
+WORKDIR /tmp/app
+ADD . ./
+
+RUN npm install -g yarn bower
+RUN yarn
+RUN bower --allow-root install
+RUN yarn run build optimize-images
+RUN yarn run build
```

Comment:
Yes. One executes a specific gulp task, the other the default one. The default one does not include optimize-images (by design)